README.md

```diff
@@ -105,15 +105,15 @@ We expect each service to have the following tags within Dynatrace:
 
 E.g., the service `carts` in the stage `dev` within project `sockshop` would have the following tags:
 
-* `keptn_project:sockshop`
-* `keptn_stage:dev`
-* `keptn_service:carts`
-* `keptn_deployment:primary` (or `keptn_deployment:canary` during tests)
+* `keptn_project.sockshop`
```

Comment:
I think you accidentally changed a lot of `.` to `:`, please review those changes.